gratipay/utils/team_history.py

```diff
@@ -0,0 +1,190 @@
+from datetime import datetime
+from decimal import Decimal as D
+
+from aspen import Response
+
+
+def get_end_of_year_totals(db, team, year):
+
+    received = db.one("""
```

Comment:
It's hard to go part-way there without going whole-hog. In general writing raw SQL works well for us, apart from a few cases like @mattbk cites.

Why bring it up in this context in particular? What are you seeing that makes you think we need an abstraction here?